scripts/eslint.py

```diff
@@ -0,0 +1,62 @@
+"""  # lint-amnesty, pylint: disable=django-not-configured
```

Comment:
```suggestion
"""  # pylint: disable=django-not-configured
```

No need to add `lint-amnesty` when disabling a pylint warning. The lint-amnesty tag is supposed to indicate pylint violations which were disabled in bulk a few years ago.